druid/src/widget/split.rs

```diff
@@ -88,38 +98,89 @@ impl<T> Split<T> {
         self.splitter_size = splitter_size;
         self
     }
+
+    /// Set the minimum width of the splitter area.
+    ///
+    /// The minimum splitter area defines the minimum width of the area
+    /// where mouse hit detection is done for the splitter.
+    /// The final area is either this or the splitter size, whichever is greater.
+    ///
+    /// This can be useful when you want to use a very narrow visual splitter,
+    /// but don't want to sacrifice user experience.
+    ///
+    /// The default minimum splitter area is `6.0`.
+    pub fn min_splitter_area(mut self, min_splitter_area: f64) -> Self {
+        assert!(
+            min_splitter_area >= 0.0,
+            "min_splitter_area must be 0.0 or greater!"
+        );
+        self.min_splitter_area = min_splitter_area;
+        self
+    }
+
     /// Set whether the splitter's split point can be changed by dragging.
     pub fn draggable(mut self, draggable: bool) -> Self {
         self.draggable = draggable;
         self
     }
+
     /// Builder-style method to set whether the splitter handle is drawn as a solid rectangle.
     ///
     /// If this is `false` (the default), it will be drawn as two parallel lines.
     pub fn fill_splitter_handle(mut self, solid: bool) -> Self {
         self.solid = solid;
         self
     }
-    fn splitter_hit_test(&self, size: Size, mouse_pos: Point) -> bool {
+
+    /// Returns the width of the area of mouse hit detection.
+    #[inline]
+    fn splitter_area(&self) -> f64 {
+        self.splitter_size.max(self.min_splitter_area)
+    }
+
+    /// Returns the padding width added to the splitter on both sides.
```

Comment:
```suggestion
    /// Returns the padding width added to the splitter on each side.
```
I think the 'each side' phrasing makes more clear that we're dividing the difference by two - we *should* add this to each side, not redivide.

Reply:
agreed.